bemoji

```diff
@@ -20,6 +20,14 @@ bm_private_mode=${BEMOJI_PRIVATE_MODE:-false}
 # Do not sort results
 bm_ignore_recent=${BEMOJI_IGNORE_RECENT:-false}
 
+declare -A default_pickers=(
+	["bemenu"]="bemenu -p 🔍 -i -l 20"
+	["wofi"]="wofi -p 🔍 -i --show dmenu"
+	["rofi"]="rofi -p 🔍 -i -dmenu --kb-custom-1 "Alt+1" --kb-custom-2 "Alt+2""
+	["dmenu"]="dmenu -p 🔍 -i -l 20"
+	["ilia"]="ilia -n -p textlist -l 🔍"
```

Comment:
From the ilia documentation I gathered that we can display our little spy-glass with the `-l` option but I have no way of testing this locally without access to ilia.

Reply:
![2023-08-25_15-43](https://github.com/marty-oehme/bemoji/assets/18191/91f1973f-91d5-462a-bf56-f9eb50ca6189)

🔍 is now displayed at the bottom, but I now think it's not quite what you would expect it to. Here are some options:

`ilia -n -p textlist -l "Emoji 🔍"`
![2023-08-25_16-18](https://github.com/marty-oehme/bemoji/assets/18191/f941bc90-505c-4290-bd40-61825a339c09)

`ilia -n -p textlist -l "🔍 Emoji"`
![2023-08-25_16-19](https://github.com/marty-oehme/bemoji/assets/18191/10337ad0-4cba-4ec6-88db-6ad1ca3ee4eb)

`ilia -n -p textlist -l "Emoji" -i desktop-magnifier` (from the GTK theme)
![2023-08-25_16-21](https://github.com/marty-oehme/bemoji/assets/18191/86fb3d16-61c0-4029-a373-7188cc5c008c)

`ilia -n -p textlist -l "🔍 Emoji" -i ""`
![2023-08-25_16-22](https://github.com/marty-oehme/bemoji/assets/18191/403dc9de-5f34-41ed-b67c-2697f831016c)

@kgilmer I might be missing something... is there a way to completely disable the icon (like with `Help`) or use a character and not a GTK icon?

Reply:
Ah, I knew I misunderstood something with the label. If there is no option to completely disable the icon I think I am fine with either of the last two choices; perhaps the GTK-icon actually makes the most sense since we are now in GTK-world. Thanks for looking into it!